plugins/importexport/native/filter/PKPAuthorNativeXmlFilter.php

```diff
@@ -128,12 +129,19 @@ public function createPKPAuthorNode($doc, $author)
         }
 
         $this->createOptionalNode($doc, $authorNode, 'country', $author->getCountry());
-        $authorNode->appendChild($doc->createElementNS($deployment->getNamespace(), 'email', htmlspecialchars($author->getEmail(), ENT_COMPAT, 'UTF-8')));
+        $this->createOptionalNode($doc, $authorNode, 'email', htmlspecialchars($author->getEmail() ?? '') ?: null);
```

Comment:
if email does not exists
htmlspecialchars($author->getEmail() ?? '') would return empty string
so the last parameter for createOptionalNode is never null.

Maybe to have it something like:
$author->getEmail() ? htmlspecialchars($author->getEmail() : null